src/main/java/com/amplitude/HttpTransport.java

```diff
@@ -263,6 +279,22 @@ private EventsRetryResult retryEventsOnce(String userId, String deviceId, List<E
           eventsToRetry.add(event);
         } else {
           eventsToDrop.add(event);
+          if (recordThrottledId) {
+            try {
+              JSONObject throttledUser = response.rateLimitBody.getJSONObject("exceededDailyQuotaUsers");
```

Comment:
The current throttling information is based on user id and device id. Is there any other cases for it? like reach day limit? How would we handle for that case?

Reply:
429 error like event per second limit or daily limit all related to  and limited by userId and deviceId. The detail error message may be different but userId/deviceId is the way to tell whether an event will be throttled.

Reply:
put the code block in the right place, also correct the json keywords from exceededDailyQuotaUsers to throttledUsers,  exceededDailyQuotaDevices to throttledDevices